[ERROR] There is a conflict between multiple exceptions for assigning the same variable.

This justification is true:
  --> test_scope/bad/scope.catala_en
   | 
14 |   definition b under condition not c consequence equals 0
   |                                ^^^^^
   + Article

This justification is true:
  --> test_scope/bad/scope.catala_en
   | 
13 |   definition b under condition not c consequence equals 1337
   |                                ^^^^^
   + Article
